releng/gradle-composite/helper.gradle

```diff
@@ -0,0 +1,22 @@
+ext.getProjectsFromLocationProperties = {
+	def result = new ArrayList<File>()
+	def propertyFile = file('../../locations.properties')
+    if (propertyFile.exists()) {
+        def Properties props = new Properties()
+        def names = XTEXT_GRADLE_PROJECT_NAMES.split(",")
+        props.load(new FileInputStream(propertyFile))
+        names.each {
+            name ->
+            def propertyName = "git.clone.xtext.${name}.location"
+            def propertyDefault = "gitclonextext${name}locationprototype"
+            def propertyValue = props.getProperty(propertyName)
+            if (propertyValue != null && propertyValue != propertyDefault) {
+            	def file = file(propertyValue)
+                if (file.exists()) {
+                	result.add(file)
+                }
+            }
+    	}
+    } 
+    return result
+}
```

Comment:
Wouldn't it be possible to extract more common functionality, e.g. the fallback to default project paths?